src/lib/preview-websocket.ts

```
@@ -0,0 +1,382 @@
+                // Simular mensaje enviado
+                const newMessage = {
+                    id: Date.now(),
+                    contenido: data.message,
+                    emisor_id: currentUser.id,
+                    receptor_id: data.recipientId,
+                    fecha_envio: new Date().toISOString(),
+                    leido: false,
+                    match_id: data.matchId
+                };
+
+                this.cache.addMessage(newMessage);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)